blobmodel/blobs.py

```diff
@@ -26,14 +28,53 @@ def __init__(
         perp_shape_parameters: dict = None,
         blob_alignment: bool = True,
     ) -> None:
+
+        """
+        Initialize a single blob.
+
+        Attributes
+        ----------
+        blob_id : int
+            Identifier for the blob.
+        blob_shape : AbstractBlobShape
+            Shape of the blob.
+        amplitude : float
+            Amplitude of the blob.
+        width_prop : float
+            Width of the blob in the propagation direction.
+        width_perp : float
+            Width of the blob in the perpendicular direction.
+        v_x : float
+            Velocity of the blob in the x-direction.
+        v_y : float
+            Velocity of the blob in the y-direction.
+        pos_x : float
+            Initial position of the blob in the x-direction.
+        pos_y : float
+            Initial position of the blob in the y-direction.
+        t_init : float
+            Initial time of the blob.
+        t_drain : Union[float, NDArray]
+            Time scale for the blob to drain.
+        prop_shape_parameters : dict
+            Additional shape parameters for the propagation direction.
+        perp_shape_parameters : dict
+            Additional shape parameters for the perpendicular direction.
+        blob_alignment : bool, optional
+            If blob_aligment == True, the blob shapes are rotated in the propagation direction of the blob
+            If blob_aligment == False, the blob shapes are independent of the propagation direction
+        theta : float
```

Comment:
Should this be documented? It is not an input parameter, maybe it should be private
In either case, if documented, it should be "Angle of the blob's velocity vector with the x-axis if blob_aligment=True, else 0"

Reply:
Thanks, I removed theta from the docstring.